audition/distance_from_best.py

```diff
@@ -225,7 +239,7 @@ def generate_plot_data(
                 )
                 SELECT dist.model_group_id, pct_diff, mg.model_type,
                        COUNT(*) AS num_models,
-                       AVG(CASE WHEN below_best <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
+                       AVG(CASE WHEN dist_from_best_case <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
```

Comment:
Probably a little annoying since I realize it comes from my initial code, but if we're fixing the naming for directionality, it seems like it makes sense to fix `pct_diff` as well since most metrics aren't necessarily expressed in terms of percentages...

This actually also brings up the problem that the `GENERATE_SERIES` between 0 and 1 isn't general enough in its own right, either. Instead, we might need to do something like infer the min and max values (maybe with a little padding). What do you think?